src/sentry/static/sentry/app/views/organizationEventsV2/loadingContainer.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+
+import LoadingIndicator from 'app/components/loadingIndicator';
+
+export default function LoadingContainer(props) {
```

Comment:
Is this a component we could generalize and use outside of events? It looks like it could be useful in more places.

Reply:
Sounds good, moved this into /components